spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -145,6 +145,26 @@
     end
   end
 
+  context 'selection is quick' do
+    it 'selects template, flavor and tenant quickly' do
+      vmts = []
+      100.times do
+        atype = create(:appliance_type)
+        t = create(:openstack_with_flavors, funds: [fund])
+        vmts << create(
+          :virtual_machine_template,
+          appliance_type: atype,
+          tenants: [t]
+        )
+      end
+      t1 = Time.now.to_f
+      subject.select_tmpl_and_flavor_and_tenant(vmts)
+      t2 = Time.now.to_f
+      expect(t2 - t1).to be < 5,
+        "Flavor selection process took longer than 5 seconds"
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.